pkgs/top-level/release.nix

```diff
@@ -12,7 +12,7 @@
 { nixpkgs ? { outPath = (import ./all-packages.nix {}).lib.cleanSource ../..; revCount = 1234; shortRev = "abcdef"; }
 , officialRelease ? false
 , # The platforms for which we build Nixpkgs.
-  supportedSystems ? [ "x86_64-linux" "i686-linux" /* "x86_64-darwin" */ ]
+  supportedSystems ? [ "x86_64-linux" "i686-linux" "x86_64-darwin" ]
```

Comment:
We shouldn't change this until we have sufficient OS X build capacity in the build farm.